### ISSUE TITLE
A new Melt widget icon

### ISSUE THREAD

**Issue by simonaubertbd** [biolab/orange3#6739]
**What's your use case?**
As of today, the melt widget icon is a shopping cart. This is misleading and not easy to understand.

**What's your proposed solution?**
A new icon, pretty much explicit.
![melt](https://github.com/biolab/orange3/assets/46959192/b7c43dbb-32a8-457c-a681-8eeba5ead95a)
https://github.com/simonaubertbd/misc_icons/blob/main/melt.svg

**Are there any alternative solutions?**
N/A

**Comment by markotoplak** [comment]
Feri's idea: melting ice cream.

Also, meting ice cube? Icebergs? :)

**Comment by simonaubertbd** [comment]
@markotoplak  why not.. but the idea with melted chocolate table... is that's a table with rows and columns ;)

**Comment by markotoplak** [comment]
@simonaubertbd, true, that is your icon. The only problem is I tend to each chocolates before they have time to melt.

**Comment by borondics** [comment]
<img width="149" alt="image" src="https://github.com/biolab/orange3/assets/6304794/b523c3e6-da54-4c1d-91d9-e6ffdb5eb971">

**Comment by janezd** [comment]
- The existing icon (shopping cart) reflects a common use case (at least in tutorials)
- The icon proposed by @simonaubertbd is interesting, but doesn't reflect what the widget is doing.
- @borondics' icon is great because it's funny, but it visualizes the name.
- @wvdvegte's tells me what the widget is doing (which I -- not being a regular pandas user -- tend to forget).

In #7234 I attempted to replicate the latter idea; more comments in the PR.

**Comment by simonaubertbd** [comment]
@janezd Hello. On Amphi we decided to rename the widget to transpose, that helps to understand (because melting... I don't follow pandas dev logic here).. and the icon is now explicit, especially beside pivot

<img width="208" height="102" alt="Image" src="https://github.com/user-attachments/assets/fd9d8e00-7088-494d-b754-093c0319b8be" />

**Comment by janezd** [comment]
I didn't like the icon on the first glance, but it indeed works great when put beside Pivot. Nice!

As for terms, I too don't like Melt. But as for "Transpose" -- Isn't it confusing that the term already carries a different meaning in mathematics?

**Comment by simonaubertbd** [comment]
This is the same term used in Excel or Alteryx. Other option would be a Pivot rows to columns/Pivot columns to rows (clearer but longer). And to be honest, I'm not sure anyone think about a matrix transposition.

**Comment by wvdvegte** [comment]
The term "melting" wasn't invented by Pandas developers, but it is more widely used. The often-cited paper ["Tidy Data" by Hadley Wickham](https://www.jstatsoft.org/article/view/v059i10/0) uses it, too, and nicely explains it in another way: it transforms a "wide table" into a "tall table". This is perhaps seen as a "melting" effect where gravity stretches that data downward once it has become "liquid".
Perhaps "wide-to-tall" would be a more expressive widget name?
There already is a Transpose widget, that does a much more straightforward table transposition (the same as "Paste Transposed" after Copy in Excel).
